src/packages/buskill/__init__.py

```diff
@@ -224,6 +227,25 @@ def __init__(self):
 		self.CURRENT_PLATFORM = platform.system().upper()
 		self.ERR_PLATFORM_NOT_SUPPORTED = 'ERROR: Your platform (' +str(platform.system())+ ') is not supported. If you believe this is an error, please file a bug report:\n\nhttps://github.com/BusKill/buskill-app/issues'
 
+		config = SafeConfigParser()
+
+		self.CONFIG = None
+
+		# NOTE self.config will change to a dictionary if the file is found 
+		# 	if not a startup routine should begin 
+		#	Dictionary keys will be the setting and the value the variable 
+		# 	i.e. Trigger would be the key, the trigger file location would be the value 
+
+		if os.file.exists(APP_DIR, 'buskill.conf'):
+			self.CONFIG = dict()
+			config.read('buskill.conf')
```

Comment:
shouldn't this be `os.path( APP_DIR, 'buskill.conf' )`?

Reply:
Hello Matthew, 
apologies I forgot I worked on this while waiting for a VM to come up. 

The code is inactive I can amend this and re push code up. 

actual feature progress is in work on a different branch on my fork 

and yes the line should be if os.file.exists(os.path(APP_DIR, 'buskill.conf'))

Reply:
sorry, I was referring to the `config.read()` line -- shouldn't that be `config.read( os.path(APP_DIR,'buskill.conf') )`?